grails-app/conf/Config.groovy

```diff
@@ -87,6 +87,9 @@ grails.hibernate.pass.readonly = false
 // configure passing read-only to OSIV session by default, requires "singleSession = false" OSIV mode
 grails.hibernate.osiv.readonly = false
 
+//TODO change me
+grails.plugin.raven.dsn = "https://{PUBLIC_KEY}:{SECRET_KEY}@app.getsentry.com/{PATH}{PROJECT_ID}"
```

Comment:
Are `PUBLIC_KEY` and `SECRET_KEY` automatically available here if they're environment variables?
I'd prefer a `SENTRY_DSN` env var because we're hosting our own sentry server, not the cloud hosted `getsentry.com`.